Modules/IxNetRestApi.py

```diff
@@ -155,7 +161,7 @@ def __init__(self, apiServerIp=None, serverIpPort=None, serverOs='windows', linu
 
         self._session = requests.Session()
 
-        self.serverOs = serverOs ;# windows|windowsConnectionMgr|linux
+        self.serverOs = serverOs  # windows|windowsConnectionMgr|linux
         self.jsonHeader = {"content-type": "application/json"}
```

Comment:
Is this required for RestPy?

Reply:
Yes Raj. This is required to set the RestPy port. Please see the example below:
 _if self.serverOs == 'windows':
            if self.apiServerPort is None:
                self.apiServerPort = 11009_

Reply:
Removed self.jsonHeader = {"content-type": "application/json"}